<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customers</class>
 <widget class="QMainWindow" name="Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>1123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Désactivé</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnaddclient">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>30</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ajouter un client</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableCustomers">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>1351</width>
      <height>781</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adresse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NPA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Localité</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mobile</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="LblMsgConfirmation">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
